File: client-python/consumerCalculadora.py
from suds.client import Client
from suds.cache import NoCache
url="http://localhost:8080/server-java/CalculadoraService?WSDL"
client = Client(url, cache=NoCache())
#invocando os metodos
print 'Soma: ', client.service.somarCalc(11,2)
print 'Soma: ', client.service.subtrair(10,12)

File: client-node/consumerCalculadora.js
var soap = require('soap');
var url = 'http://localhost:8080/server-java/CalculadoraService?WSDL';
var primeiro = {'primeiro':2, 'segundo':4};
var segundo = {arg0:9, arg1:4};
soap.WSDL.prototype.ignoredNamespaces = [];
soap.createClient(url, function(err, client) {
    client.somarCalc(primeiro, function(err,result,res){
         console.log('Método somar:',result.valor);
    })
    client.subtrair(segundo, function(err,result,res){
         console.log('Método subtrair:',result.return);
    })
});
